MesnardBarba-CiSE.md

```diff
@@ -118,26 +118,27 @@ Still, we ran against an obscure trick of the trade that changed our results com
 
 The numerical approach in IBAMR belongs to the same family as that used in our published work on wakes of flying snakes: an immersed boundary method. 
 The essence of the approach is that the fluid is represented by a fixed structured mesh, while the immersed body is represented by its own, separate mesh that moves with the body. 
-We speak of an Eulerian mesh for the fluid, and a Lagrangian mesh for the solid. 
+(word choice) an Eulerian mesh for the fluid, and a Lagrangian mesh for the solid. 
```

Comment:
It's certainly grammatically correct, it just struck me as strange sounding.  
Possibly something like
"That is, we create an Eulerian..."
?